tests/test_syntax/extensions/test_tables.py

```diff
@@ -60,3 +61,861 @@ def test_empty_cells(self):
+                <td>word 1</td>
+                <td><code>word |2</code></td>
+                <td>word 3</td>
+                </tr>
+                <tr>
+                <td>words</td>
+                <td><code>some | code</code></td>
+                <td>more words</td>
+                </tr>
+                <tr>
+                <td>words</td>
+                <td><code>some | code</code></td>
+                <td>more words</td>
+                </tr>
+                <tr>
+                <td>words</td>
+                <td><code>some | code</code></td>
+                <td>more words</td>
+                </tr>
+                <tr>
+                <td>words</td>
+                <td><code>some ` | ` code</code></td>
+                <td>more words</td>
+                </tr>
+                <tr>
+                <td>words</td>
+                <td><code>some ` | ` code</code></td>
+                <td>more words</td>
+                </tr>
+                <tr>
+                <td>words</td>
+                <td><code>some ` | ` code</code></td>
+                <td>more words</td>
+                </tr>
+                </tbody>
+                </table>
+                """
+            ),
+            extensions=['tables']
+        )
+
+    def test_issue_440(self):
+        self.assertMarkdownRenders(
+            self.dedent(
+                """
+                A test for issue #440:
+
+                foo | bar
+                --- | ---
+                foo | (`bar`) and `baz`.
+                """),
+            self.dedent(
+                """
+                <p>A test for issue #440:</p>
+                <table>
+                <thead>
+                <tr>
+                <th>foo</th>
+                <th>bar</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr>
+                <td>foo</td>
+                <td>(<code>bar</code>) and <code>baz</code>.</td>
+                </tr>
+                </tbody>
+                </table>
+                """
+            ),
+            extensions=['tables']
+        )
+
+    def test_lists_not_tables(self):
+        self.assertMarkdownRenders(
+            self.dedent(
+                """
+                Lists are not tables
+
+                 - this | should | not
+                 - be | a | table
+                """),
+            self.dedent(
+                """
+                <p>Lists are not tables</p>
+                <ul>
+                <li>this | should | not</li>
+                <li>be | a | table</li>
+                </ul>
+                """
+            ),
+            extensions=['tables']
+        )
+
+    def test_issue_449(self):
+        self.assertMarkdownRenders(
+            """
+Add tests for issue #449
+
+Odd backticks | Even backticks
+------------ | -------------
+``[!\\"\\#$%&'()*+,\\-./:;<=>?@\\[\\\\\\]^_`{|}~]`` | ``[!\\"\\#$%&'()*+,\\-./:;<=>?@\\[\\\\\\]^`_`{|}~]``
```

Comment:
This is only 107 characters long. However, we allow up to 128 character line length. There is plenty of room to indent this.

Reply:
I had indented this line before I sent it in. However, running `tox` against it blew a warning on the line if I didn't split it, which is why I pulled it back flush. I'm not sure if that's a tox problem or something else, but I figured that if `tox` was complaining I should fix it. As mentioned previously, if there weren't already so many escapes in the line, I would have quoted the newline and left it indented, but that didn't seem a good idea here.

Reply:
> if there weren't already so many escapes in the line

Maybe you can use a raw string here (`r"""`) to reduce the number of escapes?

Reply:
Good call. That resolved it. Code updated.